### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by budfox3** [LouisChen1992/Deep_Learning_in_Asset_Pricing#4]
I'm trying to implement your approach and I have a question about the estimation procedure. Is it correct that you don't use batches and always feed in the whole training set as a whole, compute the loss at t = T and backpropagate once every epoch? Thus, for the LSTMs you feed the macro data from t=1 until t=T and then repeat the LSTM output N times? Thanks & regards

**Comment by LouisChen1992** [comment]
Right. We don't use mini-batch here.

**Comment by budfox3** [comment]
Thank you very much.
